resources/views/card/access.blade.php

```diff
@@ -12,10 +12,10 @@
     <li class="list-group-item">Roden Street Door: {{ $roden }}</li>
     @endif
     <li class="list-group-item">{{ Auth::user() == $user ? 'You have' : $user->getFirstname() . ' has' }} {{ count($user->getRfidTags()) }} RFID cards.</li>
-    @can('pins.view.all')
```

Comment:
I'd sooner add a `pins.view.own` permission, add to the correct role, then test for all conditions

can view all || (can view own && set to enroll)

Reply:
Thanks - I will fix that!